src/song/mod.rs

```diff
@@ -366,6 +366,41 @@ mod tests {
         assert_eq!(FEATURES_VERSION, song.features_version);
     }
 
+    #[test]
+    #[cfg(feature = "symphonia-flac")]
+    fn test_analyze_with_symphonia() {
+        use crate::decoder::symphonia::SymphoniaDecoder;
+
+        let song =
+            SymphoniaDecoder::song_from_path(Path::new("data/s16_mono_22_5kHz.flac")).unwrap();
+        let expected_analysis = vec![
```

Comment:
would it be possible to make this vec a constant in the test module, and use it in both `test_analyze_with_symphonia` and `test_analyze`? To make it clearer that we're testing against the same set of values

Reply:
sure thing